cpp/inc/bond/protocol/compact_binary.h

```diff
@@ -86,8 +90,10 @@
                                             and then encoded as unsigned integer
 
 
-                     float, double          little endian
+                     float, double          little endian IEEE 764
```

Comment:
The size can be included too. How about "32- or 64-bit little endian IEEE 764"?

Reply:
Sounds good!